mypy/semanal.py

```diff
@@ -3546,6 +3550,11 @@ def visit_file(self, file: MypyFile, fnam: str, mod_id: str, options: Options) -
                 else:
                     typ = UnionType([UnboundType('__builtins__.str'),
                                      UnboundType('__builtins__.unicode')])
+            elif name == '__path__':
+                # __path__ is only defined on __init__.py files
+                if not os.path.basename(fnam).lower() == '__init__.py':
+                    continue
+                typ = UnboundType(t)
```

Comment:
Trying `typing.List` results in `error: Name 'typing' is not defined`. Maybe I need to include some fixture to get that definition?